specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-07-01/managedHsm.json

```diff
@@ -0,0 +1,1473 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "description": "The URI to poll for completion status.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ManagedHsm"
+            }
+          },
+          "200": {
+            "description": "Patched managed HSM Pool",
+            "schema": {
+              "$ref": "#/definitions/ManagedHsm"
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ManagedHsmError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update an existing managed HSM Pool": {
+            "$ref": "./examples/ManagedHsm_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagedHsms"
+        ],
+        "operationId": "ManagedHsms_Delete",
+        "x-ms-long-running-operation": true,
+        "description": "Deletes the specified managed HSM Pool.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagedHsmResourceGroupName"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the managed HSM Pool to delete"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "description": "The URI to poll for completion status.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
+            "description": "Delete successful."
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ManagedHsmError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a managed HSM Pool": {
+            "$ref": "./examples/ManagedHsm_Delete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ManagedHsms"
+        ],
+        "operationId": "ManagedHsms_Get",
+        "x-ms-long-running-operation": true,
```

Comment:
> "x-ms-long-running-operation": true, [](http://example.com/codeflow?start=7&length=37)

GET cannot be long running.

Reply:
After confirming with the Managed HSM team, I've removed this from the GET operation.